framework/src/main/java/org/checkerframework/common/aliasing/AliasingTransfer.java

```diff
@@ -83,40 +85,62 @@ public TransferResult<CFValue, CFStore> visitAssignment(
      */
     @Override
     protected void processPostconditions(
-            MethodInvocationNode n, CFStore store, ExecutableElement methodElement, Tree tree) {
+            Node n, CFStore store, ExecutableElement methodElement, ExpressionTree tree) {
+        // Process MethodInvocation node only
+        MethodInvocationNode methodInvocationNode = null;
+        ObjectCreationNode objectCreationNode = null;
+        if (n instanceof MethodInvocationNode) {
+            methodInvocationNode = (MethodInvocationNode) n;
+        } else if (n instanceof ObjectCreationNode) {
+            objectCreationNode = (ObjectCreationNode) n;
+        } else {
+            throw new BugInCF(
+                    "Error passing other type of node"
```

Comment:
How about: `"processPostconditions in AliasingTransfer expects a MethodInvocationNode or ObjectCreationNode argument; received a " + n.getClass().getSimpleName()` ?
Same below.